examples/functions/rfc3339_parse/function.tf

```diff
@@ -0,0 +1,11 @@
+terraform {
+  required_providers {
+    time = {
+      source = "hashicorp/time"
+    }
+  }
+}
+
+output "example_output" {
+  value = provider :: time :: rfc3339_parse("2023-07-25T23:43:16Z")
```

Comment:
The beta release of Terraform contains a `terraform fmt` fix for this, so will wait to update this example before merging.